tracer/test/test-applications/integrations/Samples.OracleMDA/Samples.OracleMDA.csproj

```diff
@@ -11,16 +11,22 @@
 
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Oracle.ManagedDataAccess" Version="$(ApiVersion)" />
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' ">
+    <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="$(ApiVersion)"/>
+  </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">
+    <PackageReference Include="Oracle.ManagedDataAccess" Version="$(ApiVersion)"/>
+  </ItemGroup>
+
+  <ItemGroup>
     <ProjectReference Include="..\dependency-libs\Samples.DatabaseHelper\Samples.DatabaseHelper.csproj" />
     <ProjectReference Include="..\dependency-libs\Samples.DatabaseHelper.netstandard\Samples.DatabaseHelper.netstandard.csproj" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="System.Security.Permissions" Version="5.0.0" />
-    <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
+    <PackageReference Include="System.Configuration.ConfigurationManager" Version="8.0.0"/>
```

Comment:
about this one, really I have no idea what I'm doing, I changed that following a [NU1605 error](https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605) and it worked, so 🤷

Reply:
This is probably because you changed the tested version significantly. If they're referenced transitively anyway , you can presumably just remove these references (or add them conditionally for .NET Framework for example)